### DISCUSSION TITLE
recover wa monitor data

### DISCUSSION THREAD

**Discussion by LouisXIVLouis** [rizwansoaib/whatsapp-monitor#186] [Q&A]
Using Chrome Browser extension 2.8 in Chromium on Linux

When the browser freezes, for example "page unresponsive" error, is there a way to find the last known good wa monitor data?

I looked in /.config/chromium/Default/Extensions/blablabla but no data there obvi

**Comment by rizwansoaib** [comment]
https://wpmonitor.tech 

You can check your data here.

**Comment by LouisXIVLouis** [comment]
Perfect. Thank you

**Reply by rizwansoaib** [reply]
Your Welcome 🙏
